src/pages/community.js

```diff
@@ -0,0 +1,676 @@
+            <FeatureImage
+              fluid={data.ethereum.childImageSharp.fluid}
+              alt={translateMessageId(
+                "page-community-open-source-image-alt",
+                intl
+              )}
+            />
+          </ImageContainer>
+        </RowReverse>
+      </OpenSourceContainer>
+      <PoapContainer>
+        <Row>
+          <FeatureContent>
+            <LeftColumnContent>
+              <H2>
+                {/* <Translation id="page-community-poap" /> */}
+                Contribute to ethereum.org
+              </H2>
+              <Subtitle>
+                {/* <Translation id="page-community-poap-description" /> */}
+                For many people, ethereum.org is their first step into the
+                ecosystem. It is kept up-to-date and accurate by thousands of
+                open-source contributors. Want to help? Read our guide on
+                contributing, or take up an issue on our GitHub.
+              </Subtitle>
+              <ButtonRow>
+                <ButtonLink to="/contributing/">
+                  {/* <Translation id="page-community-poap-button" /> */}
+                  More on contributing
+                </ButtonLink>
+                <StyledButtonLink
+                  isSecondary
+                  to="https://github.com/ethereum/ethereum-org-website/"
+                >
+                  {/* <Translation id="page-community-poap-secondary-button" /> */}
+                  View on GitHub
+                </StyledButtonLink>
+              </ButtonRow>
+            </LeftColumnContent>
+          </FeatureContent>
+          <ImageContainer>
+            <FeatureImage
+              fluid={data.finance.childImageSharp.fluid}
+              alt={translateMessageId("page-index-internet-image-alt", intl)}
+            />
+          </ImageContainer>
+        </Row>
+      </PoapContainer>
+      <SupportContainer>
+        <RowReverse>
+          <FeatureContent>
+            <H2>
+              {/* <Translation id="page-community-support" /> */}
+              Ethereum support
+            </H2>
+            <Subtitle>
+              {/* <Translation id="page-community-support-description" /> */}
+              Need support? There is no official Ethereum support, but hundreds
+              of helpful communities are available to help you prosper on
+              Ethereum.
+            </Subtitle>
+            <div>
+              <ButtonLink to="/community/support/">
+                {/* <Translation id="page-community-support-button" /> */}
+                Get support
+              </ButtonLink>
+            </div>
+          </FeatureContent>
+          <ImageContainer>
+            <FeatureImage
+              fluid={data.hackathon.childImageSharp.fluid}
+              alt={translateMessageId("page-community-support-alt", intl)}
+            />
+          </ImageContainer>
+        </RowReverse>
+      </SupportContainer>
+      <Content>
```

Comment:
To me I think this is redundant when we have a dedicated events page, and link out to it above this section.

Reply:
[Removed](https://github.com/ethereum/ethereum-org-website/pull/4521/commits/4d77177c2234e509e0a908d53ae0b5e9ac80afb7) and replaced it with some [call outs to other pages](https://github.com/ethereum/ethereum-org-website/pull/4521/commits/c564ab28c60dec35d01b2f1193535f84d02924f9)